mindjunkies/courses/models.py

```
@@ -165,8 +168,9 @@ class Module(BaseModel):
     details = models.TextField()
     course = models.ForeignKey(Course, on_delete=models.CASCADE, related_name="modules")
     order = models.PositiveIntegerField(default=0)
+    progression = models.PositiveIntegerField(default=0)
 
-    class Meta:
+    class Meta: 
```

Ruff (0.8.2)

Trailing whitespace

Remove trailing whitespace

(W291)